OptimoveSDK/Tests/Sources/Optimobile/AnalyticsHelperTests.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Test.swift
+//  Optimove
+//
+//  Created by Kostya Antipochkin on 2024-11-22.
+//
+
+import XCTest
+@testable import OptimoveSDK
+import OptimoveTest
+
+class AnalyticsHelperTests: XCTestCase {
+    
+    var mockHttpClient: MockKSHttpClient!
+    var analyticsHelper: AnalyticsHelper!
+    
+    override func setUp() {
+        super.setUp()
+        mockHttpClient = MockKSHttpClient()
+        analyticsHelper = AnalyticsHelper(httpClient: mockHttpClient)
+    }
+    
+    override func tearDown()
+    {
+        analyticsHelper = nil
+        mockHttpClient = nil
+        super.tearDown()
+    }
+    
+    func test_number_of_sent_events_same_as_tracked() {
+        let numberOfEvents = 8;
```

Comment:
I know this is super random and unnecessary, but there are a bunch of semicolons we might not need hahah (3 in this file)

Reply:
:sweat_smile: fixed